client/remoterepo.c

```diff
@@ -924,7 +924,7 @@ TDNFDownloadPackage(
 
     if(!nSilent)
```

Comment:
Unneeded now. We can also remove getting the value for `nSilent` and eliminate the variable in this function.